src/sentry/web/api.py

```diff
@@ -267,7 +267,17 @@ class StoreView(APIView):
 
     """
     def post(self, request, **kwargs):
-        data = request.body
+        try:
+            data = request.body
+        except Exception:
```

Comment:
could we be explicit here? otherwise I think we should log the exact error that happened